src/components/AddNewIdea/AddNewIdea.test..js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
Your file has two `..` in the name, maybe a typo?

Reply:
yes! Thank you)))